src/cvode/cvode.c

```diff
@@ -3390,9 +3392,7 @@ static int cvDoErrorTest(CVodeMem cv_mem, int* nflagPtr, sunrealtype saved_t,
 
   dsm = cv_mem->cv_acnrm * cv_mem->cv_tq[2];
 
-  SUNLogDebug(CV_LOGGER, "error-test",
-              "step = %li, h = " SUN_FORMAT_G ", dsm = " SUN_FORMAT_G,
-              cv_mem->cv_nst, cv_mem->cv_h, dsm);
+  SUNLogDebug(CV_LOGGER, "error-test", "dsm = " SUN_FORMAT_G, dsm);
```

Comment:
I'm surprised that this debug line removed information -- was the previous one incompatible with the current `tools/suntools/logs.py`?

Reply:
The step number and size are logged at the beginning of the step attempt so I removed them from here so they're not duplicated in the output. Also we print `nst + 1`, so this was off by one when I was debugging.